chall/str_replace.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+#include <string>
+using namespace std;
+
+string ReplaceString(string subject, const string& search, const string& replace) {
+	size_t pos = 0;
+	while ((pos = subject.find(search, pos)) != std::string::npos) {
```

Comment:
You area already using namespace std. Why not "string::npos" instead?

Reply:
Good catch. Fixed.